libs/core/langchain_core/utils/function_calling.py

```diff
@@ -295,14 +295,12 @@ def convert_to_openai_function(
         k in function for k in ("name", "description", "parameters")
     ):
         return function
-    # a JSON schema with title and description
-    elif isinstance(function, dict) and all(
-        k in function for k in ("title", "description", "properties")
-    ):
+    # a JSON schema
+    elif isinstance(function, dict) and "properties" in function:
         function = function.copy()
         return {
-            "name": function.pop("title"),
-            "description": function.pop("description"),
+            "name": function.pop("title", "extract"),
+            "description": function.pop("description", ""),
```

Comment:
This would be OK by me if we were to omit the key, but if we're subbing in a default value of `""` I don't know that consistent, now it actually feels like we're interpreting the API to make something that used to be a requirement of the API to making it optional in our API?